datajoint/heading.py

```diff
@@ -191,70 +198,46 @@ def init_from_database(self, conn, database, table_name):
                     attr['dtype'] = numeric_types[(t, is_unsigned)]
         self.attributes = OrderedDict([(q['name'], Attribute(**q)) for q in attributes])
 
-    def proj(self, attribute_list, renamed_attributes, include_primary_key):
+    def project(self, attribute_list, named_attributes, force_primary_key=None):
```

Comment:
Sorry. Always run into this error. I got confused because it shows a deleted line `proj` above it.